### PR TITLE
Install MySQL in development environment.

### LINKED ISSUE #451: Install MySQL in development environment.
Changes to allow the installation of MySQL in the KubeArchive development environment.

### PR DESCRIPTION
**Pull request by ggallen**
## Which Issue(s) this Pull Request Resolves

Resolves #451

## Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by maruiz93 — CHANGES_REQUESTED** [review]
Please add the copyright header in the different files

**Comment by ggallen** [comment]
> Please add the copyright header in the different files

@maruiz93 , I added the copyright to `install.sh`.

File: integrations/database/mysql/mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (beerparty) [inline integrations/database/mysql/mysql.yaml:1]: Should our YAML files have copyright? I see some YAML files in our code have copyright info at the top, others do not.

**Review by maruiz93 — APPROVED** [review]

File: integrations/database/mysql/mysql.yaml (hunk 1 above)
Comment (ggallen) [reply integrations/database/mysql/mysql.yaml:1]: @beerparty , I don't see the copyright in any YAML files.

**Review by beerparty — APPROVED** [review]

**Pull request merged**